tests/util/src/opentelemetry/test/test_base.py

```diff
@@ -44,11 +46,11 @@ def tearDownClass(cls):
     def setUp(self):
         self.memory_exporter.clear()
 
-    def check_span_instrumentation_info(self, span, module):
+    def assertSpanInstrumentationInfo(self, span, module):
```

Comment:
Mentioned this in the contrib repo PR, but I would prefer the name `assertSpanHasInstrumentationInfo`

Reply:
Updated